aca_preview/preview.py

```diff
@@ -136,10 +142,13 @@ def make_starcat_plot(self):
 
         stars = StarsTable.from_agasc(self.att, date=self.date)
         self.stars = stars
+        self.acqs.stars = stars
+        # Explicitly set bad_stars so that plot_stars does not apply its
+        # internal version (which is not always the right answer).
+        self.acqs.bad_stars = np.zeros(len(stars), dtype=bool)
 
         fig = plt.figure(figsize=(4.5, 4))
         ax = fig.add_subplot(1, 1, 1)
-        self.acqs.stars
```

Comment:
Not sure if this was doing something behind the scenes, but I moved it up and set it to share `stars` above.